client.go

```diff
@@ -149,9 +156,9 @@ func (client *Client) setupTransport() {
 
 	if transport == nil {
 		if client.options.Dsn == "" {
-			transport = new(noopTransport)
+			transport = &noopTransport{Logger: client.Logger}
```

Comment:
Let the rest of the SDK use the global `Logger` for now, we solve that later. Tracked by #182.